.github/workflows/test-engine-container.yml

```
@@ -17,7 +17,7 @@ on:
         required: true
 
 env:
-  IMAGE_NAME: ${{ vars.DOCKERHUB_USERNAME }}/voicevox_engine
+  IMAGE_NAME: ghcr.io/${{ github.repository_owner }}/voicevox_engine
```

ここ、本当は多分dockerhubから持ってくるテストをしたい気はしますね！
となると`ghcr.io`にしてる理由がレートリミットのためだと書いておかないと間違えて書き換えてしまいそう！

ということでコメント書いておくのどうでしょう！
```suggestion
  IMAGE_NAME: ghcr.io/${{ github.repository_owner }}/voicevox_engine # NOTE: Docker Hubのレートリミット回避のためGitHub Container Registryを使う
```

---

なるほどです。ユーザーが使うのは基本Docker Hubの方だと思うので、本当にテストしたいのはDocker Hubな気はしますね。

Suggestion適用します。